tests/vmi_lifecycle_test.go

```diff
@@ -364,11 +347,10 @@ var _ = Describe("[Serial][rfe_id:273][crit:high][vendor:cnv-qe@redhat.com][leve
 						}
 					}
 					By("Starting a VirtualMachineInstance")
-					_, err := virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmi)
-					Expect(err).To(BeNil(), "Should create VMI successfully")
+					vmi = tests.RunVMIAndExpectScheduling(vmi, 30)
```

Comment:
Changes to this test file also seem to be unrelated in this commit.

Reply:
The test failed on parallel runs without this. The `findLauncherForVMI` was not suitable for parallel runs. I could have fixed the function, but I decided to use the `tests.GetPodByVirtualMachineInstance` function which  we already have. Since this function expected that the pod for the VMI already exists (which `findLauncherForVMI` did not expecte), I also started to use `tests.RunVMIAndExpectScheduling`.

Reply:
Thank you for the clarification.